plugins/arDominionB5Plugin/modules/browse/templates/hierarchySuccess.php

```diff
@@ -0,0 +1,20 @@
+<?php decorate_with('layout_1col.php'); ?>
+<?php slot('title'); ?>
+
+  <div class="d-flex justify-content-end flex-wrap gap-2 ms-auto">
+    <input type="button" id="fullwidth-treeview-reset-button" class="btn btn-sm atom-btn-white" value="<?php echo __('Reset'); ?>" />
+    <input type="button" id="fullwidth-treeview-more-button" class="btn btn-sm atom-btn-white" data-label="<?php echo __('%1% more'); ?>" value="" />
+  </div>
+
+  <?php echo image_tag('/vendor/jstree/themes/default/throbber.gif', ['id' => 'fullwidth-treeview-activity-indicator', 'alt' => __('Loading ...')]); ?>
+  <h1><?php echo __('Hierarchy'); ?></h1>
```

Comment:
I'd change the order here too and use the same loading spinner we use in other places. For example ...

https://github.com/artefactual/atom/blob/dev-test/issue-13530-b5-themes/plugins/arDominionB5Plugin/modules/user/templates/indexSuccess.mod_standard.php#L97-L100

Something like this ...

```html
<h1>
  <?php echo __('Hierarchy'); ?>
  <span id="fullwidth-treeview-activity-indicator">
    <i class="fas fa-spinner fa-spin ms-2" aria-hidden="true"></i>
    <span class="visually-hidden"><?php echo __('Loading ...'); ?></span>
  </span>
</h1>
```